model-fcrepo/src/main/java/edu/unc/lib/boxc/model/fcrepo/services/RepositoryObjectDriver.java

```diff
@@ -171,7 +171,8 @@ public InputStream getBinaryStream(BinaryObject obj) throws FedoraException {
         PID pid = obj.getPid();
 
         try {
-            FcrepoResponse response = getClient().get(pid.getRepositoryUri()).perform();
```

Comment:
I'd vote for reverting the changes in this file since it's just reformatting at this point